### PR TITLE
chore: Translations for the labels taxonomy

### PR DESCRIPTION
**Pull request by teolemon**
Translations for the labels taxonomy

### CODE REVIEW

File: taxonomies/labels.txt [hunk 1]
```diff
@@ -318,6 +320,7 @@ es:Producto carbono neutro
 fi:hiilihyvitetty tuote, hiilikompensoitu tuote, hiilineutraali
 fr:Produit compensé carbone, Carbone neutre, neutre en carbone, carbone compensé
 hu:szénsemleges termék, szénsemleges, Szén-kompenzált termék
+it:Prodotto compensato al carbonio, carbon neutral
```
Comment (VaiTon) [inline taxonomies/labels.txt:323]: Never heard of "Prodotto compensato al carbonio".
```suggestion
it:Impatto zero, Carbon neutral
```

File: taxonomies/labels.txt [hunk 2]
```diff
@@ -2583,7 +2598,9 @@ es:Sin potenciador artificial del sabor
 fi:ei keinotekoisia arominvahventeita, ilman keinotekoisia arominvahventeita
 fr:Sans exhausteurs de goût artificiels
 hu:Mesterséges ízfokozó nélkül, mesterséges ízfokozót nem tartalmaz
+it:Nessun esaltatore di sapidità artificiale
```
Comment (VaiTon) [inline taxonomies/labels.txt:2601]:
```suggestion
it:Senza esaltatori artificiali di sapidità
```

File: taxonomies/labels.txt [hunk 3]
```diff
@@ -17368,6 +17441,7 @@ protected_name_type:en: aoc
 en:with Guérande salt
 ca:Amb sal de Guérande
 fr:Au sel de Guérande
+it:con sale di Guérande
```
Comment (VaiTon) [inline taxonomies/labels.txt:17444]:
```suggestion
it:Con sale di Guérande
```

File: taxonomies/labels.txt [hunk 4]
```diff
@@ -17602,6 +17676,7 @@ ingredients:en: en:pork
 <fr:Viande Française
 en:French beef
 fr:Viande Bovine Française, VBF, Viande bovine Origine France, Bœuf 100% Français, Viande de bœuf Française, Bœuf 100% France, Bœuf Origine France
+it:Manzo francese
```
Comment (VaiTon) [inline taxonomies/labels.txt:17679]:
```suggestion
it:Manzo di origine francese, manzo francese
```

File: taxonomies/labels.txt [hunk 5]
```diff
@@ -17616,6 +17691,7 @@ origins:en: en:france
 <fr:Viande Française
 en:French poultry
 fr:Volaille Française, volaille Origine France
+it:Pollame francese
```
Comment (VaiTon) [inline taxonomies/labels.txt:17694]:
```suggestion
it:Pollame di origine francese, pollame francese
```

File: taxonomies/labels.txt [hunk 6]
```diff
@@ -18479,8 +18557,10 @@ hu:Új
 it:Nuovo
 nl:Nieuw
 pl:Nowość, Nowy, Nowe
+pt:Novo
 ru:Новинка
 th:สินค้าใหม่
+tr:Yeni
 
 en:New recipe
```
Comment (VaiTon) [inline taxonomies/labels.txt:18565]: Why new recipe and nutriscore?

**Review by VaiTon — CHANGES_REQUESTED** [review]
Checked IT

File: taxonomies/labels.txt [hunk 7]
```diff
@@ -3168,17 +3191,19 @@ ca:Orgànic,de cultiu ecologic
...
-es:Ecológico, ecológica, Producto ecológico, Biológico, Producto biológico, Orgánico, Producto orgánico, Eco, Bio, Ingredientes ecológicos, Ingrediente ecológico, ingredientes procedentes de la agricultura ecológica, ingrediente procedente de la agricultura ecológica, agricultura ecológica, cumple con el reglamento de agricultura ecológica CE 2092/91, procedentes de la agricultura ecológica, procedente de la agricultura ecológica,de agricultura ecológica,de agricultura ecologica, Procedente e agricultura ecológica
+es:Ecológico, ecológica, Producto ecológico, Biológico, Producto biológico, Orgánico, Producto orgánico, Eco, Bio, Ingredientes ecológicos, Ingrediente ecológico, ingredientes procedentes de la agricultura ecológica, ingrediente procedente de la agricultura ecológica, agricultura ecológica, cumple con el reglamento de agricultura ecológica CE 2092/91, procedentes de la agricultura ecológica, procedente de la agricultura ecológica,de agricultura ecológica,de agricultura ecologica
```
Comment (stephanegigandet) [inline taxonomies/labels.txt:3194]: Procedente e agricultura ecológica was added recently, it needs to be added back.

File: taxonomies/labels.txt (hunk 6 above)
Comment (teolemon) [reply taxonomies/labels.txt:18565]: sometimes product change, and new recipe is displayed on the packaging

File: taxonomies/labels.txt (hunk 7 above)
Comment (teolemon) [inline taxonomies/labels.txt:3194]:
```suggestion
es:Ecológico, ecológica, Producto ecológico, Biológico, Producto biológico, Orgánico, Producto orgánico, Eco, Bio, Ingredientes ecológicos, Ingrediente ecológico, ingredientes procedentes de la agricultura ecológica, ingrediente procedente de la agricultura ecológica, agricultura ecológica, cumple con el reglamento de agricultura ecológica CE 2092/91, procedentes de la agricultura ecológica, procedente de la agricultura ecológica,de agricultura ecológica,de agricultura ecologica, Procedente e agricultura ecológica
```

File: taxonomies/labels.txt (hunk 1 above)
Comment (teolemon) [inline taxonomies/labels.txt:323]:
```suggestion
it:Impatto zero, Carbon neutral, Prodotto compensato al carbonio
```

File: taxonomies/labels.txt (hunk 7 above)
Comment (teolemon) [inline taxonomies/labels.txt]:
```suggestion
es:Ecológico, ecológica, Producto ecológico, Biológico, Producto biológico, Orgánico, Producto orgánico, Eco, Bio, Ingredientes ecológicos, Ingrediente ecológico, ingredientes procedentes de la agricultura ecológica, ingrediente procedente de la agricultura ecológica, agricultura ecológica, cumple con el reglamento de agricultura ecológica CE 2092/91, procedentes de la agricultura ecológica, procedente de la agricultura ecológica,de agricultura ecológica,de agricultura ecologica, Procedente e agricultura ecológica
```

File: taxonomies/labels.txt (hunk 4 above)
Comment (teolemon) [inline taxonomies/labels.txt]:
```suggestion
it:Manzo di origine francese, manzo francese
```

File: taxonomies/labels.txt (hunk 5 above)
Comment (teolemon) [inline taxonomies/labels.txt]:
```suggestion
it:Pollame di origine francese, pollame francese
```

**Comment by alexgarel** [comment]
@VaiTon can you verify if it's ok now ?

**Comment by benbenben2** [comment]
@stephanegigandet, this pr has been "git merged" with branch main and conflicts have been resolved as well. If you can approve it, we can squash and merge

**Pull request merged**